ccd-definition/ComplexTypes/2_ChildParty.json

```diff
@@ -273,5 +273,14 @@
     "ElementLabel": "Give details, including assessment outcomes and referrals to health services",
     "FieldShowCondition": "litigationIssues=\"YES\"",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ChildParty",
+    "ListElementCode": "showAddressInConfidentialTab",
+    "FieldType": "Text",
```

Comment:
Could this be a YesOrNo? We could then use the YesNo enum, although its values may need to be annotated with the yes and no in the appropriate format.

Reply:
is it really necessary? It will only ever be populated with yes when making confidential

Reply:
Probably not, just suggesting possibilities, we could then make it clear in the code what the value is expecting

Reply:
so that's the thing, this value will never be read by java code and it will never be no. So I fail to see the benefits of updating this